src/components/SearchPane/_search-pane.scss

```diff
@@ -0,0 +1,210 @@
+
+// Variables
+
+$transition-duration: 0.4s;
+
+// Extends
+
+%btn-nav-base {
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  font-size: var(--text-sm);
+  height: 64px;
+  justify-content: flex-start;
+  letter-spacing: 0.025rem;
+  min-height: 64px;
+  padding: 0 calc(var(--space-unit) * 4);
+}
+
+// Styles
+
+.search-pane {
+  background: var(--colour-white);
+  bottom: 0;
+  left: 0;
+  position: fixed;
+  right: 0;
+  top: 0;
+  transform: translate3d(100%, 0, 0);
+  transition: transform ease $transition-duration;
+  z-index: 4;
+
+  @include breakpoint(md) {
+    bottom: initial;
+    position: absolute;
+    transform: translate3d(0, -100%, 0);
+  }
+
+  &.is-active {
+    transform: translate3d(0, 0, 0);
+
+    @include breakpoint(md) {
+      transform: none;
+    }
+  }
+}
+
+.search-pane__content {
+  align-items: center;
+  background: var(--colour-white);
+  box-sizing: border-box;
+  display: flex;
+  height: 100%;
+  justify-content: stretch;
+  padding: var(--space-lg);
+  position: relative;
+  transition: transform ease $transition-duration;
+  width: 100%;
+  z-index: 1;
+
+  @include breakpoint(sm) {
+    padding-left: var(--space-xl);
+    padding-right: var(--space-xl);
+  }
+
+  @include breakpoint(md) {
+    min-height: 64px;
+    padding: 0;
+  }
+
+  &.is-active {
+    box-shadow: var(--space-xs) 0 var(--space-md) rgba(var(--colour-black), 0.1);
+  }
+}
+
+.search-pane__container {
+  margin: 0 auto;
+  max-width: 1344px;
+  width: 100%;
+
+  @include breakpoint(md) {
+    align-items: center;
+    display: flex;
+  }
+}
+
+.search-pane__controls {
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+
+  @include breakpoint(md) {
+    margin-left: var(--space-xl);
+    position: static;
+  }
+}
+
+.search-pane__back {
+  @extend %btn-nav-base;
+
+  border-top: 1px solid var(--colour-grey-05);
+  color: var(--colour-blue-80);
+  width: 100%;
+
+  @include breakpoint(md) {
+    display: none;
+  }
+}
+
+.search-pane__back .icon {
+  display: flex;
+  justify-content: flex-end;
+  margin-right: var(--space-xs);
+  transform: rotate(180deg);
+}
+
+.search-pane__close {
+  @extend %btn-nav-base;
+
+  color: var(--colour-grey-60);
+}
+
+.search-pane__close .icon {
+  margin-right: var(--space-md);
+}
+
+.search-pane__form {
+  align-items: center;
+  display: flex;
+  justify-content: stretch;
+  width: 100%;
+}
+
+.search-pane__form-label {
+  flex: auto;
+}
+
+.search-pane__form-input {
+  background: var(--colour-grey-05);
+  padding: var(--space-md) var(--space-lg);
+  width: 100%;
+
+  @include breakpoint(md) {
+    background: var(--colour-white);
+  }
+}
+
+.search-pane__form-submit {
+  transition: opacity ease $transition-duration;
+
+  @media screen and (max-width: (map-get($breakpoints, md))) {
+    padding: 0;
+  }
+
+  @include breakpoint(md) {
+    color: var(--colour-white);
+    margin-right: var(--space-lg);
+  }
+}
+
+// .search-pane__form-submit .btn__text {
```

Comment:
remove if no longer needed

Reply:
is needed - however I'm going to remove and add in separate PR (as then will be able to use visually-hidden mixin/extend 🚀)